source/about/mattermost-v9-changelog.md

```diff
@@ -22,65 +22,73 @@ Support for Mattermost Server v8.1 [Extended Support Release](https://docs.matte
 If you upgrade from a release earlier than v9.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
 ```
 
-### Improvements
-
-See [this walkthrough video](https://mattermost.com/video/mattermost-v9-10-changelog/) on some of the improvements in our latest release below.
-
-#### User Interface (UI)
- - Pre-packaged GitLab plugin version [v1.9.1](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.9.1).
- - Pre-packaged Mattermost Copilot plugin version [v0.8.1](https://github.com/mattermost/mattermost-plugin-ai/releases/tag/v0.8.1).
- - Pre-packaged Calls plugin version [v0.28.2](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.28.2).
- - Re-designed the user profile popover and improved its performance.
- - Added banner to prompt users to give desktop notification permissions when opening the app.
- - Increased the width of the profile picture setting to match other user settings.
- - Improved screen reader support for the emoji picker.
- - Improved the accessibility of plugin buttons in the channel header.
+```{tab} Improvements
```

Comment:
I remembered this previous conversation about the importance of being able to use search https://hub.mattermost.com/private-core/pl/7x4fgsy3itbn7kf3hqf8akhfoo. Will the search functionality be affected?

Reply:
Really great point, @amyblais! Let's try it out and see what the search experience will be like.

Reply:
I did some testing and I'm unable to get search results from any closed tabs. Maybe we can add Ben and Matt to this PR review for any feedback as well from user experience perspective.

Reply:
I've included both @hanzei & @matthewbirtch as reviewers on this PR and share their thoughts on the importance of changelog detail searchability in our docs when the content is moved into tabs.

Reply:
I've also reached out to @lindy65 to see if the QA Community can run some tests to determine whether tabbed content negatively impacts search. If it's determined through testing that search isn't impacted (it's no better nor any worse), that may provide the confidence we need to move forward. (I'd still want to add a task to our Docs backlog to review the capabilities of the Sphinx search engine to make site search better overall).

Reply:
To clarify: My concerns were (and are) about the browser search, i.e., search via ctrl + f, not about the custom search function that we built.

On the preview environment, I could not search for content in a tag. For example, searching for `visibilitychange` did not return a result until I opened "Bug Fixes".

Are there changelog pages from other software products from which we can draw inspiration? Are there ones that use tabs?

Reply:
Thank you for clarifying the problem space. Using content tabs is uncommon in changelogs and release notes since the "hidden" content isn't searchable nor accessible for SEO purposes. Moving forward with a content tab approach would require us to ensure all tab content is loaded in the DOM and searchable. It would be worth exploring whether the Sphinx extensions we're using for content tab display support this.

One alternate way forward could be expandable/collapsible content -- especially if an "expand all" option were provided for page-level searching.

Reply:
Agreed, @matthewbirtch.